Samples/src/Teleportation/TeleportationSample.qs

```diff
@@ -47,21 +47,21 @@ namespace Microsoft.Quantum.Samples.Teleportation {
             let data1 = M(msg);
             let data2 = M(register);
 
-            // decode the message by applying the corrections on
+            // Decode the message by applying the corrections on
             // the target qubit accordingly:
-            if (data1 == One) { Z(target); }
-            if (data2 == One) { X(target); }
+            if (M(msg)      == One) { Z(target); }
```

Comment:
Possible use for `IsResultOne` library function.

Reply:
I think I especially like this one, as we can demonstrate both ways of comparing results, show off Q# a little bit.